xtp/CMakeLists.txt

```diff
@@ -43,6 +43,11 @@ if(USE_CUDA)
   set_package_properties(CUDA PROPERTIES TYPE REQUIRED PURPOSE "Used for GPU acceleration")
 endif(USE_CUDA)
 
+find_package(Python 3 COMPONENTS Interpreter Development)
```

Comment:
I think we are looking for the python unconditionally in the root cmake now.

Reply:
removed